### DISCUSSION TITLE
求助：backend ip saved in master does not equal to backend local ip

### DISCUSSION THREAD

**Discussion by suran44** [apache/doris#17399] [A - General / Q&A]
有两台服务器192.168.1.2（部署了docker FE）以及服务器192.168.1.3（部署了docker BE），请问要怎么在192.168.1.2这台FE添加BE呢alter system add backend，如果是设置192.168.1.3 的BE的实际docker ip地址172.18.0.3，报无法连接172.18.0.3，设置成192.168.1.3，报错backend ip saved in master does not equal to backend local ipXXXX, actual backend local ip，请问一下这个要怎么办呢

**Comment by FreeOnePlus** [comment]
可以使用 host 模式来完成

**Comment by FreeOnePlus** [comment]
如有帮助需要，可加微信 fl_manyi 来给你指导
